validation/superchain-version_test.go

```diff
@@ -30,29 +56,18 @@ func testContractsMatchATag(t *testing.T, chain *ChainConfig) {
 
 	versions, err := getContractVersionsFromChain(*Addresses[chain.ChainID], client)
 	require.NoError(t, err)
-	_, err = findOPContractTag(versions)
+	_, err = findOPContractTagInVersions(versions)
+	require.NoError(t, err)
+
+	bytecodeHashes, err := getContractBytecodeHashesFromChain(chain.ChainID, *Addresses[chain.ChainID], client)
+	require.NoError(t, err)
+	_, err = findOPContractTagInByteCodeHashes(L1ContractBytecodeHashes(bytecodeHashes))
```

Comment:
Is this type conversion necessary?

Reply:
Not needed, actually don't remember why I had to add it in the first place.